src/Sylius/Behat/Page/Shop/Checkout/SelectShippingPage.php

```diff
@@ -95,7 +95,9 @@ public function getItemSubtotal($itemName)
 
     public function nextStep()
     {
-        $this->getElement('next_step')->press();
+        if (!$this->getElement('next_step')->hasClass('disabled')) {
```

Comment:
After [this](https://github.com/Sylius/Sylius/pull/6075#discussion_r80230316) it can be removed.